manager/integration/tests/test_settings.py

```diff
@@ -1241,3 +1242,58 @@ def test_setting_update_with_invalid_value_via_configmap(core_api, request):  #
                        ""])
 
     cleanup_volume_by_name(client, vol_name)
+
+
+def test_setting_v1_data_engine(client, request): # NOQA
+    """
+    Test that the v1 data engine setting works correctly.
+    1. Create a volume and attach it.
+    2. Set v1 data engine setting to false. The setting should be rejected.
+    3. Detach the volume.
+    4. Set v1 data engine setting to false again. The setting should be
```

Comment:
You can add one more step between 4 and 5. After disabling the v1 engine, attach the volume and it should fail.

Reply:
Added. Thank you.